rc-files/download_resources.sh

```diff
@@ -0,0 +1,11 @@
+# Get the latest stable release and unzip
+wget "https://github.com/Moo-Ack-Productions/MCprep/releases/download/3.5.3/MCprep_addon_3.5.3.zip"
```

Comment:
Should either be a TODO or maybe even just a $1 input flag for tag version.

Could be fancier and use something like `git describe --tags --abbrev=0` to just get the latest tag and directly use that. The hard coding I just think will get overlooked is all